setup.cfg

```diff
@@ -14,13 +14,13 @@ markers =
 [pep8]
 max-line-length = 99
 ignore = E121,E123,E126,E133,E226,E241,E242,E302,E704
-exclude = build/*,.tox/*,tests/*,ve/*,*/_vendor/*,auxlib/*,conda/progressbar/*,conda/compat.py
+exclude = build/*,.tox/*,tests/*,ve/*,*/_vendor/*,auxlib/*,conda/progressbar/*,conda/compat.py,utils/win_batlink.py
 
 
 [flake8]
 max-line-length = 99
 ignore = E121,E123,E126,E133,E226,E241,E242,E302,E704
-exclude = build/*,.tox/*,tests/*,ve/*,*/_vendor/*,auxlib/*,conda/progressbar/*,conda/compat.py
+exclude = build/*,.tox/*,tests/*,ve/*,*/_vendor/*,auxlib/*,conda/progressbar/*,conda/compat.py,utils/win_batlink.py
```

Comment:
What exactly is `utils/win_batlink.py` for? The very first line is `# UNUSED MODULE`.